### DISCUSSION TITLE
Is it possible to use OBDLink SX with SavvyCAN ?

### DISCUSSION THREAD

**Discussion by J1850** [collin80/SavvyCAN#682] [Q&A]
Hello Collin80.
Is it possible to use SavvyCAN with OBDLinkSX ? I am trying to monitir the data from the J1850PWM bus on my car.
I am trying to see what codes change with an event.
Any help would be appreciated.
Thanks. Pete

**Comment by collin80** [comment]
Currently SavvyCAN is truly CAN only. It partially supports CAN-FD but LIN, K-Line, and other protocols aren't really supported. So, J1850 PWM isn't going to really map into the software properly. Also, I don't think there is currently any way to make an OBDLinkSX work with SavvyCAN. There is a very buggy and broken implementation of LAWICEL in SavvyCAN and that could have maybe supported the OBDLinkSX but it's too broken to really be usable.

**Reply by J1850** [reply]
Thanks Collin80. I will try a different software and hope to solve the issue. All the best. Peter
